### DISCUSSION TITLE
...

### DISCUSSION THREAD

**Discussion by glowsoony** [KadeArchive/Kade-Engine#2528] [General]
.._. .. -. -.. .... .. --

**Comment by isakube** [comment]
shut yo ARG-lookin ass up lmao

**Comment by isakube** [comment]
but if this ends up being an ARG i WILL try to solve it

**Reply by glowsoony** [reply]
- .-. -.---

**Reply by glowsoony** [reply]
ekaw pu siht si ton a ekoj ekawer10inaga30ekawer21

**Reply by ghost** [reply]
what

**Reply by isakube** [reply]
> * .-. -.---

tfw the arg writer can't speak morse code correctly

**Reply by glowsoony** [reply]
its a fucking date and the words are back words NO  HINTS!

**Reply by glowsoony** [reply]
WILL BE RELEAVED AT 10/26/21

**Reply by isakube** [reply]
"no hints"

*proceeds to state the release date*

**Reply by glowsoony** [reply]
> ekaw pu siht si ton a ekoj ekawaer10inaga30ekawaer21

this is backwords SO TRY AND NO HINTS BUT THERE IS A SECRECT TO THIS! BECAUSE SOMETHING IN THAT DATE IS GOING TO HAPPEN!

**Reply by glowsoony** [reply]
ALR

**Reply by glowsoony** [reply]
number 1 is mores code: f i n d h i m

**Reply by glowsoony** [reply]
2nd: try

**Reply by glowsoony** [reply]
3th: w a k e u p t h i s i s n o t a j o k e r e a w a k e a g a i n R E A W A K E
spells: wake up this is not a joke reawake again RE A WAKE
date: 10/30/21 this was hidden in that message

**Reply by isakube** [reply]
cring
